includes/course-theme/class-sensei-course-theme-templates.php

```diff
@@ -236,6 +380,12 @@ public function get_block_templates() {
 		$templates    = [];
 
 		foreach ( $this->file_templates as $name => $template ) {
+			// Prefill the template contents from their content files.
+			if ( isset( $template['content'] ) && file_exists( $template['content'] ) ) {
+				// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents -- Local file usage.
+				$template['content'] = file_get_contents( $template['content'] );
```

Comment:
Could not come up with how to sanitize this. I tried `esc_html` but that breaks the templates. Could you hint me which sanitizer I could use for this?

> check that it's really a local file?

You mean a file that's inside a WP installation? Doesn't the `file_exists()` check means the file is a local file vs url?

Reply:
Right, I think `file_exists` ensures that. (Because the reason of the PHPCS check is that `file_get_contents` works with URLs too, not just file paths)